cm/sw_update_fom.c

```diff
@@ -123,8 +123,6 @@ static int cm_swu_fom_tick(struct m0_fom *fom)
 			M0_LOG(M0_ERROR, "SWU phase=%d rc=%d.", phase, rc);
 		else
 			M0_LOG(M0_DEBUG, "SWU phase=%d rc=%d.", phase, rc);
-
-		m0_cm_sw_remote_update(cm);
```

Comment:
Why remove this? This is required in case rc is M0_FSO_WAIT/M0_FSO_AGAIN

Reply:
It is already being called under the following block of 'if' statement.

Reply:
> It is already being called under the following block of 'if' statement.

OK, I see that we can have a non-zero positive return value in which case we need this statement. Will add it back.